assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/track_predict.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""MLflow PythonModel wrapper class that loads the MLflow model, preprocess inputs and performs inference."""
+
+import re
+import subprocess
+import sys
+from typing import Dict, List
+
+import mlflow
+import pandas as pd
+import torch
+from config import MMDetLiterals, MLflowSchemaLiterals, MOTLiterals
+
+
+def _is_valid_url(text: str) -> bool:
```

Comment:
thanks, changed :) previously I think of putting all the functions inside the same file to avoid since there's only one function needs to be imported. but you are right, I can totally re-use it.

Reply:
Thanks for updating